app/controllers/page_controller.rb

```diff
@@ -77,10 +77,10 @@ def create
         process_uploaded_file(@page, @page.base_image)
       end
 
-      if subaction == 'save_and_new'
-        ajax_redirect_to({ :controller => 'dashboard', :action => 'owner', :anchor => 'create-work' })
-      else
+      if subaction == 'save_and_next'
```

Comment:
So (unfortunately) we need to undo the removal of "save and new".  

It doesn't make sense in the web UI, but it does in the mobile UI where you are taking pictures of i.e. letters page by page and adding them to an existing work.  When you reach the end of a letter you want to create a new work for the next one.  

(I'm sorry, that's a product management research problem, not a code problem.)